.protolint.yaml

```diff
@@ -0,0 +1,149 @@
+---
```

Comment:
We can probably remove this file and just use the default (no `.protolint.yaml`) to start - unless there were specific rules in here you had changed

Reply:
I did change some rules. They really don't like the single word RPC names, and they require a comment on every enum field. Both of which I find annoying.

I also temporarily disabled the requirement for a comment on every field, which I would like to turn back on once the dust settles on the definitions.